README.md

```diff
@@ -57,6 +58,22 @@ client = Bitreserve::Client.new
 puts client.all_tickers
 ```
 
+# Options
+
+This is a summary of the supported options when instantiating a new client, and their default values:
+
+```ruby
+Bitreserve::Client.new(
+  # bearer_token for OAuth authentication
+  token: ENV['BITRESERVE_AUTH_TOKEN']
+
+  # use the sandbox api (https://api-sandbox.bitreserve.org) instead of the real one
+  sandbox: false
```

Comment:
The ideal thing to do was to not have globals at all

I just hit a wall with the api_base url, because HTTParty stores that at class level
My idea was to leave that one global (at least for now, although we'll hardly need to have two simultaenous clients to different apis)

About actual pros and cons, in this specific case (and considering only the current options), I can't think of neither of them, except for the fact that this version is already done